CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # Changelog
 
+## [0.17.0](https://github.com/nodejs/gyp-next/compare/gyp-next-v0.16.2...gyp-next-v0.17.0) (2024-03-11)
```

Comment:
Looks like something's wrong in the config. Tags shouldn't include the package's name.

Reply:
/cc @lukekarrys